src/main/java/net/snowflake/client/core/SessionUtil.java

```diff
@@ -381,6 +381,7 @@ private static SFLoginOutput newSession(
                 loginInput.getUserName());
 
         loginInput.setToken(s.issueJwtToken());
+        logger.debug("JWT: {}", loginInput.getToken());
```

Comment:
@sfc-gh-mknister do you think we can just log the contents of the token rather than logging the entire signed token?

Reply:
That's pretty much what this is doing, I think! The token gets deserialized into a string value, which is what is getting returned here. I think the idea is that then Support can use the string value to see if the JWT token is valid here: https://jwt.io/. Is this okay from a security standpoint?

Reply:
As per our conversation earlier today, changed to display token info instead of raw token string!